manta-accounting/src/transfer/mod.rs

```diff
@@ -139,7 +145,8 @@ pub trait Configuration {
 
     /// Key Agreement Scheme Variable Type
     type KeyAgreementSchemeVar: Constant<Self::Compiler, Type = Self::KeyAgreementScheme>
-        + key::agreement::Types<SecretKey = SecretKeyVar<Self>, PublicKey = PublicKeyVar<Self>>
+        + key::agreement::SecretKeyType<SecretKey = SecretKeyVar<Self>>
```

Comment:
Because we're using an implementation of `Hybrid<KeyAgreementScheme, _>` which requires `EphemeralPublicKey = PublicKey` and `EphemeralSecretKey = SecretKey`. There's no such implementation for `KeyAgreementSchemeVar`. In any case, the transfer `Configuration` trait will change in the next PR.